fourmolu.yaml

```diff
@@ -7,8 +7,18 @@ respectful: true # don't be too opinionated about newlines etc.
 haddock-style: single-line # '--' vs. '{-'
 haddock-style-module: single-line
 newlines-between-decls: 1 # number of newlines between top-level declarations
-fixities: []
+fixities:
+  - infixr 0  <?>
+  - infixl 4  <@>
+  - infixl 5  ^^^
+  - infixr 5  <+>
```

Comment:
`<+>`, `<@>`, and `<.>` are all from upstream packages, right? You shouldnt need to add their fixities here

Reply:
If I eliminate `<@>`, the diff gets longer. Is this expected?

Reply:
Oh sorry I meant `<?>`

Reply:
https://github.com/haskell/cabal/blob/31ae312de34f848d7c653cbcef75e35d7fbab166/Cabal-syntax/src/Distribution/Compat/Parsing.hs#L212